modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -1,27 +1,70 @@
-= Anypoint Monitoring Performance, Impact, and Limits
+= Anypoint Monitoring Performance Impact, Limits, and Data Retention
+
+Anypoint Monitoring processes and retains massive amounts of data with a low performance impact. Titanium subscribers receive an enhanced data retention package and more storage than Gold and Platinum subscribers. For better performance and readability, your log data is truncated into short lines.
+
+ You can also improve your Anypoint Monitoring experience by adjusting your screen resolution. 
 
-Review details about the performance impact and data retention with Anypoint Monitoring.
 
 == Mule Monitoring Agent Performance Impact
 
-Based on typical usage, the expected CPU utilization is less than 5%. For more information, refer to xref:monitoring::configure-monitoring-cloudhub#performance-impact[Configure and Enable Anypoint Monitoring for Mule Apps (CloudHub)].
+Based on typical use, the expected CPU utilization from the Mule monitoring agent is less than 5%. For more information, refer to xref:monitoring::configure-monitoring-cloudhub#performance-impact[Configure and Enable Anypoint Monitoring for Mule Apps (CloudHub)].
 
-== Data Retention and Aggregation
+Anypoint Monitoring asynchronously collects data at a resolution of one minute, which does not block Mule runtime data processing.
 
-Anypoint Monitoring asynchronously collects data, which does not block Mule runtime data processing.
+== Data Retention and Limits
 
-* Minute Resolution for Metrics.
-* Gold and Platinum subscriptions:
- ** API analytics metrics data are retained for 90 days.
- ** Application metrics data are retained for 30 days.
+=== Gold and Platinum Subscription Limits
 
+Gold and Platinum subscribers to Anypoint Platform receive modest data retention and feature limits:
 
-== Screen Resolution
+Logs::
+Log limits are documented in xref:runtime-manager::viewing-log-data.adoc#log-persistence[Runtime Manager].
+
+Application metrics::
+30 days.
+
+Enhanced API metrics::
+30 days. The limit for data in the *Client Applications* tab is 1 hour.
+
+Basic alerts::
+50 basic alerts.
+
+=== Titanium Subscription Limits
+
+Titanium subscribers to Anypoint Platform receive more robust data retention and feature limits:
+
+Logs::
+In raw format, your logs can occupy as much space as you receive with your Titanium subscription. Note that the storage space you receive with your Titanium subscription is used to store logs, metrics data, and search indexes. Because indexes and metrics do not take up much storage capacity, you can use the majority of your storage for raw log data.
```

Comment:
```suggestion
In raw format, your logs can occupy as much space as you receive with your Titanium subscription. Note that the storage space you receive with your Titanium subscription is used to store logs and metrics data. Because metrics do not take up much storage capacity, you can use the majority of your storage for raw log data.
```